Rubberduck.Core/Common/RubberduckHooks.cs

```diff
@@ -13,18 +13,17 @@ namespace Rubberduck.Common
 {
     public class RubberduckHooks : SubclassingWindow, IRubberduckHooks
     {
-        private readonly AutoCompleteService _autoCompleteHook;
         private readonly IGeneralConfigService _config;
         private readonly HotkeyFactory _hotkeyFactory;
         private readonly IList<IAttachable> _hooks = new List<IAttachable>();
         private static readonly Logger Logger = LogManager.GetCurrentClassLogger();
 
-        public RubberduckHooks(IVBE vbe, IGeneralConfigService config, HotkeyFactory hotkeyFactory, AutoCompleteService autoCompleteHook)
+        public RubberduckHooks(IVBE vbe, IGeneralConfigService config, HotkeyFactory hotkeyFactory, AutoCompleteService autoComplete)
             : base((IntPtr)vbe.MainWindow.HWnd, (IntPtr)vbe.MainWindow.HWnd)
         {
-            _autoCompleteHook = autoCompleteHook;
             _config = config;
             _hotkeyFactory = hotkeyFactory;
+            AutoComplete = autoComplete;
```

Comment:
consistency is king... we might want to get the other private backing fields into properties as well